apis/v1alpha1/rhmi_types.go

```diff
@@ -78,7 +78,7 @@ var (
 	OperatorVersionRHSSO          OperatorVersion = "7.6.7-1"
 	OperatorVersionRHSSOUser      OperatorVersion = "7.6.7-1"
 	OperatorVersionCloudResources OperatorVersion = "1.1.3"
-	OperatorVersion3Scale         OperatorVersion = "0.11.8-mas"
+	OperatorVersion3Scale         OperatorVersion = "0.12.1-mas"
```

Comment:
@valerymo why are we doing this?

Reply:
Done. "0.12.1-mas" related changed removed.

Reply:
@valerymo it looks like it's not reverted, please undo the change

Reply:
sorry, updated. Thank you